lib/Checksum.py

```diff
@@ -0,0 +1,102 @@
+import base64
+import string
+import random
+import hashlib
+
+from Crypto.Cipher import AES
+
+IV = "@@@@&&&&####$$$$"
+BLOCK_SIZE = 16
+
+
+def generate_checksum(param_dict, merchant_key, salt=None):
```

Comment:
Add comments before every library function specifying, what is the purpose of the function in one line, then what are the params passed into it with there data types and also mention what is the returned data with its data type.